### ISSUE TITLE
fail gracefully with multi label.  

### ISSUE THREAD

**Issue by johnsanterre** [dssg/eights#73]
nieve run of sklearn.datasets.iris creates this errror.

M = sklearn.datasets.load_iris().data
labels = sklearn.datasets.load_iris().target
M = cast_np_nd_to_sa(M)
exp = run_std_classifiers(M,labels)
exp.make_csv()

**Comment by johnsanterre** [comment]
Really, we should also allow multi label.  This is rather important.
